main.go

```diff
@@ -107,7 +107,6 @@ func main() {
 		setupLog.Error(err, "Unable to create controller", "controller", failedCtrl)
 		os.Exit(1)
 	}
-	//+kubebuilder:scaffold:builder
```

Comment:
Did you mean to delete this line?

Reply:
No, I may have deleted it with the the changes that the operator made to main.go